client/src/components/Survey.jsx

```diff
@@ -156,7 +150,7 @@ class Survey extends React.Component {
               placeholder={'Type your SAT Score here'} />
           </div>
         </div>
-        <div className="container-fluid">
+        <div className="row">
           <div className="col-md-12">
```

Comment:
for future updates maybe consider breaking checkboxes into 1 column on small viewports but definitely not higher priority than your current list :)